tests/Twilio/Unit/Jwt/AccessTokenTest.php

```diff
@@ -97,6 +98,26 @@ public function testChatGrant(): void {
         $this->assertEquals('IS123', $grants['chat']['service_sid']);
     }
 
+    public function testPlaybackGrant(): void {
+        $scat = new AccessToken(self::ACCOUNT_SID, self::SIGNING_KEY_SID, 'secret');
+        $grant = new PlaybackGrant();
+        $playbackGrant = array('requestCredentials' => NULL,
```

Comment:
Can replace with the [short array syntax](https://www.php.net/manual/en/language.types.array.php)

```php
$playbackGrant = ["requestCredentials" => NULL, ..]
```